### PR TITLE
fix(wexond) circumvent security measures from Google

### PR DESCRIPTION
**Pull request by lmancel**
- alter user agent for most requests
- do not unnecessarily alter request and response
- ensure most background url are not undefined sometimes favicons do not exist -> do not use "undefined" as value...
- refactor: better url and title update
- chore: View does not extend BrowserView anymore https://github.com/electron/electron/issues/23#issuecomment-19613241
- chore: some typing/linting

### CODE REVIEW

File: src/wexond/renderer/app/components/Bubble/index.tsx [hunk 1]
```diff
@@ -25,18 +27,19 @@ export const Bubble = ({
...
-  maxLines?: number;
-  iconSize?: number;
-  onClick?: (e?: React.MouseEvent<HTMLDivElement>) => void;
-  onCloseClick?: (e?: React.MouseEvent<HTMLDivElement>) => void;
-  itemsPerRow?: number;
-  disabled?: boolean;
-  toggled?: boolean;
-}) => {
-  const width = getSize(itemsPerRow);
+  children?: any
```
Comment (Azevedo-Dani) [inline src/wexond/renderer/app/components/Bubble/index.tsx]:
```suggestion
  children?: JSX.Element
```

OR 
```suggestion
 export const Bubble = React.FC<{icon?: string
  invert?: boolean
  maxLines?: number
  iconSize?: number
  onClick?: (e?: React.MouseEvent<HTMLDivElement>) => void
  onCloseClick?: (e?: React.MouseEvent<HTMLDivElement>) => void
  itemsPerRow?: number
  disabled?: boolean
  toggled?: boolean}> = ....
```

File: src/wexond/renderer/app/models/tab.ts [hunk 2]
```diff
@@ -144,48 +144,45 @@ export class Tab {
...
 
-        this.title = title
-        this.url = url
+        this.url = newUrl
 
         this.updateData()
       },
     )
 
+    ipcRenderer.on(`view-title-updated-${this.id}`, (e: any, title: string) => {
```
Comment (Azevedo-Dani) [inline src/wexond/renderer/app/models/tab.ts]: Seems not used
```suggestion
    ipcRenderer.on(`view-title-updated-${this.id}`, (_, title: string) => {
```

**Review by Azevedo-Dani — APPROVED** [review]
just some remarks related to types. not mandatory
LGTM

File: src/wexond/renderer/app/components/Bubble/index.tsx [hunk 3]
```diff
@@ -1,18 +1,20 @@
-import * as React from 'react';
-import { StyledBubble, Title, Icon, Circle, StyledClose } from './style';
-import { observer } from 'mobx-react';
+import * as React from 'react'
+import { StyledBubble, Title, Icon, Circle, StyledClose } from './style'
+import { observer } from 'mobx-react'
+import ensureIcon from '../../utils/ensureIcon'
+
+export const getSize = (i: number): number => {
+  const width = 800
```
Comment (Azevedo-Dani) [inline src/wexond/renderer/app/components/Bubble/index.tsx:7]: why 800 there's a reason ?

Comment (Azevedo-Dani) [reply src/wexond/renderer/app/components/Bubble/index.tsx:7]: I saw it was like that before my bad

**Review by Azevedo-Dani — APPROVED** [review]

File: src/wexond/renderer/app/components/Bubble/index.tsx (hunk 1 above)
Comment (lmancel) [reply src/wexond/renderer/app/components/Bubble/index.tsx]: React.FC does not seem to be available in this version

**Pull request merged**